.github/workflows/npm-publish.yml

```
@@ -0,0 +1,74 @@
+        with:
+          fetch-depth: 0
+      
+      # Set up node version
+      - name: Setup Node js version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      
+      # Load yarn cache
+      - name: Load Yarn cache
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)